i18n/en.i18n.json

```diff
@@ -216,6 +216,7 @@
   "Message_AllowEditing_BlockEditInMinutes" : "Block Message Editing After (n) Minutes",
   "Message_AllowEditing_BlockEditInMinutesDescription" : "Enter 0 to disable blocking",
   "Message_AudioRecorderEnabled" : "Audio Recorder Enabled",
+  "Message_AudioRecorderEnabledDescription" : "For this to work upload of Wav files must be enabled in the 'File Upload' section",
```

Comment:
Thought it was a compromise to point people in the right direction as to why the recorder button had disappeared.